### PR TITLE
[LBSD-2712] - Live streams get paused

### PR DESCRIPTION
**Pull request by BrianMwangi21**
### Purpose

This PR fixes a bug that stops live streams from auto playing when loaded.

### What has changed

- Added `autoplay=1&mute=1` to the Youtube video URL to autoplay and mute the video. 
- Added `autoplay` to the iframe allow attributes

### Steps to test

1. Navigate to the Editor's interface.
2. Click "Add content here" and select Embed. Add a link to a live Youtube video and publish
3. Click "Live" to see the Live Preview. You will see the posted video which should then autoplay.

_Note: The video will autoplay when muted. This is because most modern web browsers have implemented strict policies around autoplaying videos in an effort to improve the user experience and provide greater control of playback to viewers._

Resolves: [LBSD-2712](https://sofab.atlassian.net/browse/LBSD-2712)

[LBSD-2712]: https://sofab.atlassian.net/browse/LBSD-2712?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: server/liveblog/themes/themes_assets/default/templates/template-embed-providers.html [hunk 1]
```diff
@@ -66,6 +66,19 @@
...
+            iframe.src = autoplayUrl;
+
+            var allowAttr = iframe.getAttribute('allow');
+            if (allowAttr) {
+                if (!allowAttr.includes('autoplay')) {
+                    iframe.setAttribute('allow', allowAttr + ' autoplay');
+                }
+            } else {
+                iframe.setAttribute('allow', 'autoplay');
+            }
```
Comment (eos87) [inline server/liveblog/themes/themes_assets/default/templates/template-embed-providers.html:80]: nit?:
```suggestion
            iframe.setAttribute('allow', 'autoplay');
            if (allowAttr && !allowAttr.includes('autoplay'))
                iframe.setAttribute('allow', allowAttr + ' autoplay');
```

**Review by eos87 — CHANGES_REQUESTED** [review]
There is a problem with this approach. If the user has more than one video in the timeline, it will make them all of them autoplay right after loading the embed. I think what is needed is that if the user is playing a video and a new update arrives (automated updates) the playing progress you be kept where it is and the video you continue playing without interruptions. @aleksandarjelicic please confirm if that's the expected behaviour. Thanks!

**Comment by aleksandarjelicic** [comment]
I confirm @eos87 proposal.

**Comment by BrianMwangi21** [comment]
I have updated the PR with the proposed changes. @aleksandarjelicic @eos87

**Review by eos87 — CHANGES_REQUESTED** [review]
This works but only in the Editorial tab. If you play a video and then a new update arrives, the  green button is shown at the top of the timeline to alert the user about a new update. When you click the button, it works pretty well. Nevertheless, when you change the order to "Newest first" for example and play the video once again, if a new update arrives, there is no notification (which would be fine if the `All updates are auto-applied periodically` is on in theme settings) but the video stops the playing progress. 

Please check it and let me know if you need me to clarify something.

**Review by eos87 — APPROVED** [review]
Looks good and works great! Nice job!

**Pull request merged**